low-level/files/opt/seagate/sspl/conf/sspl.post-install.tmpl.1-node

```diff
@@ -25,3 +25,4 @@ storage_enclosure:
         port: TMPL_SECONDARY_CONTROLLER_PORT
       type: TMPL_CONTROLLER_TYPE
       user: TMPL_CONTROLLER_USER
+    type: TMPL_ENCLOSURE_TYPE
```

Comment:
@cdeshmukh 
FYI, we do validation on storage type in resource_map/lr2/storage.py
Discovery response is expected after post_install stage, so we need this for storage type check.

Reply:
Ok, Provisioner needs to supply this, if they dont, validate will fail

Reply:
Yes, currently provisioner supply this.
Referred: https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys

Reply:
Then good